docs/user/library/coverage/imageio.rst

```diff
@@ -52,16 +54,83 @@ file formats:
 * RPF TOC
 * SRP (ASRP/USRP) formats
 
-The ImageIO-EXT website includes ready to use ImageIO-Ext binaries including GDAL libraries. See
-"Release Information" for more info about the list of supported formats and available drivers.
+To make use of gdal-bridge functionality:
+
+* gdal must be installed
```

Comment:
It's called GDAL, from [their website](https://gdal.org/): "GDAL is a translator library for raster and vector geospatial data formats ..."

```suggestion
* GDAL must be installed
```
It's also OGR instead of ogc, and PROJ instead of proj.

Can you fix the other referces to it (as in, where it's used as text in a description, I think we can keep the variable and module names as they are :-D )